data/csp/azure/sle12/packages.yaml

```diff
@@ -30,3 +30,6 @@ packages:
   _namespace_azure_regionsrv_client_addon:
     package:
       - cloud-regionsrv-client-addon-azure
+  _namepsace_azure_dracut_config:
+    package:
+      - microsoft-dracut-config
```

Comment:
Is it okay to make the changes for sle12 even if it's in LTSS? That way we can clean up the overlay files in the recipes.

Reply:
We cannot keep it. If the SLE 12 image description needs to be re-generated for some reason it would get a package that is not available in the build project and repositories so we'd end up breaking the build. We will have to carry the overlay files for 3 more years until SLE 12 reaches EOL.

Reply:
I doesn't really matter. We've never used the SLE 12 recipes.

Reply:
@rjschwei since we don't use the SLE 12 recipes is it okay to leave the changes? Otherwise I'll revert the second commit and drop the SLE 12 changes

Reply:
OK, then we'll leave it. I am unaware of anyone else using this to build their own SLE 12 images. Thus we'll cross that bridge if it does become an issue.